### PR TITLE
Update ecology.md

### PR DESCRIPTION
**Pull request by mindjuice**

### CODE REVIEW

**Comment by boygirl** [comment]
@mindjuice Thanks for the edits. I want to keep links free of special characters, though, so the change to [`x` and `y`] and [`width` and `height`] cannot be merged.

**Comment by mindjuice** [comment]
I was a bit worried about that actually.  I can undo those changes. :-)

File: src/screens/docs/components/victory-errorbar/ecology.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ The `events` prop takes an array of event objects. Event objects are composed of
 
 ### eventKey
 
-The `eventKey` prop is used to assign eventKeys to data. This prop operates identically to the `x` and `y` data accessor props. By default, the eventKey of each datum will be equal to its index in the data array. `eventKey` may also be defined directly on each data object.
+The `eventKey` prop is used to assign 'eventKeys' to data. This prop operates identically to the `x` and `y` data accessor props. By default, the 'eventKey' of each datum will be equal to its index in the data array. `eventKey` may also be defined directly on each data object.
```
Comment (boygirl) [inline src/screens/docs/components/victory-errorbar/ecology.md]: I think these should be back ticks rather than single quotes

**Comment by boygirl** [comment]
@mindjuice thanks for all the proofreading help!

File: src/screens/docs/components/victory-errorbar/ecology.md (hunk 1 above)
Comment (chrisbolin) [inline src/screens/docs/components/victory-errorbar/ecology.md]: hey @mindjuice! thanks for your help on this. I'd love to merge this in. Do you have time to make those back ticks?

**Pull request merged**